### ISSUE TITLE
Global burden of colorectal cancer: emerging trends, risk factors and prevention strategies

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#37]
TL;DR

Review provides a comprehensive summary of CRC epidemiology, with emphasis on modifiable lifestyle and nutritional factors, chemoprevention and screening. Overall, the optimal reduction of CRC incidence and mortality will require concerted efforts to reduce modifiable risk factors, to leverage chemoprevention research and to promote population-wide and targeted screening.

Link 

https://www.nature.com/articles/s41575-019-0189-8#Abs1

Author/Institution

NaNa Keum (Harvard T.H. Chan School of Public Health, Boston, MA, USA)

Overview

- Certain global genetic and epigenetic aberrations are disproportionally distributed across the colorectum, which corresponds to aetiological heterogeneity of colorectal cancer (CRC), especially hypermutated cancers, by anatomical location

- With increasing incidence of CRC at younger ages, there is an urgent need to better identify high-risk individuals younger than 50 years, the age when screening typically starts

- The constellation of factors associated with westernization, such as obesity, physical inactivity, poor diets, alcohol drinking and smoking, is likely to drive increasing CRC incidence in economically transitioning countries

- Evidence indicates that aspirin probably confers chemopreventive benefit against CRC, though recommendation for its widespread prophylactic use is currently premature

- Screening colonoscopy and faecal occult blood test, when implemented appropriately per national financial and medical resources and CRC incidence, could contribute to secondary prevention of CRC

The optimal reduction of CRC incidence and mortality will require concerted efforts to reduce modifiable risk factors, to leverage chemoprevention research and to promote population-wide and targeted screening

Cite

Keum, N., Giovannucci, E. Global burden of colorectal cancer: emerging trends, risk factors and prevention strategies. Nat Rev Gastroenterol Hepatol 16, 713–732 (2019). https://doi.org/10.1038/s41575-019-0189-8

Comments
